datafusion/sqllogictest/test_files/predicates.slt

```diff
@@ -515,7 +515,7 @@ statement ok
 CREATE EXTERNAL TABLE data_index_bloom_encoding_stats STORED AS PARQUET LOCATION '../../parquet-testing/data/data_index_bloom_encoding_stats.parquet';
 
 statement ok
-set datafusion.execution.parquet.bloom_filter_enabled=true;
+set datafusion.execution.parquet.bloom_filter_on_read_enabled=true;
```

Comment:
It seems like this test was to ensure we had end to end coverage, so now that we switched the default the meaning / coverage has changed

Perhaps we can change it so:
1. verify the setting with `show datafusion.execution.parquet.bloom_filter_on_read_enabled`
2. Add a second set of queries with `set datafusion.execution.parquet.bloom_filter_on_read_enabled=false`

Reply:
Done